tasks-by-users/Tauqeer-Ahmed-99/01-First Webpage/css/styles.css

```diff
@@ -3,235 +3,212 @@ Created, maintained & Owned by: Tauqeer Khan
 Date: 16/07/21
 ******************************************/
 
-body{
-    font-family: arial;
-    margin: 10px auto;
-    background-color: lightgray;
-    
+body {
+  font-family: arial;
+  margin: 10px auto;
+  background-color: lightgray;
 }
 
-#Header{
-    width: 1000px;
-    margin: 20px auto;
-    background: white;    
-
+#Header {
+  width: 1000px;
+  margin: 20px auto;
+  background: white;
 }
 
-#main-heading h1{
-    font-family: italic;
-    font-weight: bolder;
+#main-heading h1 {
+  font-family: italic;
+  font-weight: bolder;
 }
 
-
-.first{
-    background: #181818;
-    padding: 10px;
-    
+.first {
+  background: #181818;
+  padding: 10px;
 }
 
-ul.first{
-    text-align: right;
-    list-style-type: none;
-    margin-right: 0px;
+ul.first {
+  text-align: right;
+  list-style-type: none;
+  margin-right: 0px;
 }
 
-.first li{
-    display: inline-block;
-    margin-left: 30px;
-    
+.first li {
+  display: inline-block;
+  margin-left: 30px;
 }
 
-.first a{
-    color: white;
-    text-decoration: none;
-    text-transform: uppercase;
-    font-size: 18px;
-    padding: 6px;
+.first a {
+  color: white;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 18px;
+  padding: 6px;
 }
 
-a.current{
-    background: #8f746c;
-    border-radius: 5px;
-    
+a.current {
+  background: #8f746c;
+  border-radius: 5px;
 }
-.first a:hover{
-    
-    font-weight: bolder;
-    border: 1px white solid;
-    box-shadow: 1px 1px 2px 1px rgba(250,250,250,0.8)
+.first a:hover {
+  font-weight: bolder;
+  border: 1px white solid;
+  box-shadow: 1px 1px 2px 1px rgba(250, 250, 250, 0.8);
 }
 
-#content{
-    text-align: center;
-    font-family: italic;
-    font-weight: bolder;
-    color: darkgrey;
+#content {
+  text-align: center;
+  font-family: italic;
+  font-weight: bolder;
+  color: darkgrey;
 }
 
-#Banner img{
-    width: 100%;
-    height: 500px}
+#Banner img {
+  width: 100%;
+  height: 500px;
+}
 
-#Banner{
-    
-    background-color: lightgray;
-    
-    
+#Banner {
+  background-color: lightgray;
 }
-.span-list{
-    text-align: center;
-    background-color: blanchedalmond
-        width: 100%;
-        margin: 0px auto;
-    
+.span-list {
+  text-align: center;
+  background-color: blanchedalmond;
+  width: 100%;
+  margin: 0px auto;
 }
-.span-list > h1{
-    font-family: italic;
-    font-weight: bolder;
+.span-list > h1 {
+  font-family: italic;
+  font-weight: bolder;
 }
 
-.span-list a img{
-    display: block;
-    height: 140px;
-    width: 300px;
-    margin: 10px auto;
-    padding: 20px auto;
-    
+.span-list a img {
+  display: block;
+  height: 140px;
+  width: 300px;
+  margin: 10px auto;
+  padding: 20px auto;
```

Comment:
again grp the similar properties @Tauqeer-Ahmed-99

Reply:
yep, I will consider these changes, these projects were coded when we were learning CSS and were not good at it. @coder-mezab